orchagent/portsorch.cpp

```diff
@@ -728,6 +726,33 @@ bool PortsOrch::getQueueType(sai_object_id_t queue_id, string &type)
     return true;
 }
 
+bool PortsOrch::setPortAutoNeg(sai_object_id_t id, int an)
+{
+    SWSS_LOG_ENTER();
+
+    sai_attribute_t attr;
+    attr.id = SAI_PORT_ATTR_AUTO_NEG_MODE;
+    switch(an) {
+      case 0: // leave it as is?
+        return true;
+      case 1:
+        attr.value.booldata = true;
+        break;
+      case 2:
+        attr.value.booldata = false;
+        break;
+    }
```

Comment:
why not just bool? 0, 1 here?

Reply:
The original idea was to support default/on/off. Default would imply SDK defaults. But SAI does not support that. The use case is to default to different settings based on the type of cable/transcievers/etc. I can revert this back to boolean, if you feel it won't be supported in the near future.

Reply:
according to SAI, it seems is bool value, and the default value is false. If you feel that the default value is incorrect, then we should change in the sai spec. 

I think it will be clear here to follow SAI standard.